apps/docs/src/pages/components/banner.mdx

```diff
@@ -0,0 +1,125 @@
+import { css } from "@emotion/css";
+import { bannerClasses, HvBanner } from "@hitachivantara/uikit-react-core";
+
+import { Description } from "@docs/components/Description";
+import { Page } from "@docs/components/Page";
+
+import Playground from "./Playground";
+
+import { getComponentData } from "../../utils/utils";
+
+export const getStaticProps = async ({ params }) => {
+  const meta = await getComponentData("Banner", "core", bannerClasses);
+  return { props: { ssg: { meta: meta } } };
+};
+
+<Description />
+
+<Page>
+
+<Playground
+  Component={HvBanner}
+  componentName="HvBanner"
+  controls={{
+    variant: {
+      defaultValue: "default",
+    },
+    showIcon: {
+      defaultValue: true,
+    },
+  }}
+  componentProps={{
+    open: true,
+    label: "This is an informational message.",
+    style: { position: "relative", top: 0 },
+  }}
+/>
+
+### With actions
+
+You can add a set of actions to the banner by specifying the `actions` prop. The `onAction` prop is a callback function that is called when an action is clicked.
+
+```jsx live
+<HvBanner
+  open
+  offset={0}
+  label="This is a banner."
+  showIcon
+  actions={[
+    { id: "action_1", label: "Action 1", disabled: false },
+    { id: "action_2", label: "Action 2", disabled: false },
+  ]}
+  onAction={(event, action) => console.log("Clicked", action)}
+  style={{ position: "relative", top: 0 }}
+/>
+```
+
+### With custom icons
```

Comment:
Can we get rid of the "With " prefixes? They don't add any value (any sample could be prefixed with "With " and the meaning is the same)

Reply:
Yes, I'll remove those.